databento/historical/http.py

```diff
@@ -185,7 +179,7 @@ def _stream(
     async def _stream_async(
         self,
         url: str,
-        params: List[Tuple[str, Optional[str]]],
+        params: Sequence[Tuple[str, Optional[str]]],
```

Comment:
Is there a particular reason that we want `List` in other places but a `Sequence` will suffice here? (I know Python type hints can be annoying sometimes)